public/app/features/dashboard-scene/scene/Scopes/ScopesFiltersAdvancedSelector.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SceneComponentProps } from '@grafana/scenes';
+import { Button, Drawer, Spinner, useStyles2 } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+
+import { ScopesFiltersScene } from './ScopesFiltersScene';
+import { ScopesTreeLevel } from './ScopesTreeLevel';
+
+export function ScopesFiltersAdvancedSelector({ model }: SceneComponentProps<ScopesFiltersScene>) {
+  const styles = useStyles2(getStyles);
+  const { nodes, loadingNodeName, dirtyScopeNames, isLoadingScopes, isAdvancedOpened } = model.useState();
```

Comment:
Both this object and the simple scopes selector preserves the state when closed - is this intentional? I mean, when you browse scopes and eventually drill down to some nested one, then close the scopes picker, when opening it back again it will be where you left it when closing. Not saying this is bad, but my question is more is this is something that is required (cc @Ijin08 ).